### ISSUE TITLE
studio(a11y): main menu has not enough contrast

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#554]
### Affected part of DeckDeckGo
- [x] Studio
- [ ] Remote control
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ ] Poll
- [ ] Web Components
- [ ] Templates

### Reproduction

Steps to reproduce:
1.  Go to https://deckdeckgo.com/
2. Click the main menu
 
 
### Expected Behavior

The contrast between font color and background color should be high enough.

### Actual Behavior

<img width="419" alt="DeckDeckGo main menu" src="https://user-images.githubusercontent.com/1763537/71412923-264c5f80-2650-11ea-95ef-00bfb15e20c1.png">

When checking above colors on their contrast ratio, it fails.

<img width="611" alt="Who Can Use contrast" src="https://user-images.githubusercontent.com/1763537/71412951-53007700-2650-11ea-9254-7960928f3083.png">

The provided contrast fails for basically everyone, according to [Who Can Use](https://whocanuse.com/?b=3880ff&c=FFFFFF&f=14&s=). It may be hard to solve this problem, since `#3880ff` is part of DeckDeckGo's identity.

### Environment

- Browser(s): Chrome, latest version
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by peterpeterparker** [comment]
Fully agree, probably the same with the dark mode menu aka both light and dark mode!

Thx for pinging us about this 👍

**Comment by donroyco** [comment]
@peterpeterparker Anytime. While playing with WCU, I saw that [changing the font-size to 19px and making it bold](http://whocanuse.com/?b=3880ff&c=FFFFFF&f=19&s=b), the WCAG grading goes to AA. Not sure if that makes it prettier, it makes it a least more useful.

**Comment by peterpeterparker** [comment]
@donroyco contrast will be better I hope with the PR I just merged:

light mode:

<img width="915" alt="Capture d’écran 2020-01-10 à 10 01 00" src="https://user-images.githubusercontent.com/16886711/72145778-7be87200-339b-11ea-8a6f-c169c2ae217d.png">

dark mode:

<img width="915" alt="Capture d’écran 2020-01-10 à 10 00 55" src="https://user-images.githubusercontent.com/16886711/72145791-8276e980-339b-11ea-940b-ed6b192b7557.png">

**Comment by peterpeterparker** [comment]
Thx for the issue 👍

**Comment by donroyco** [comment]
Thanks for the improvement @peterpeterparker !

**Comment by peterpeterparker** [comment]
😃
